libraries/rush-lib/src/logic/operations/ConsoleTimelinePlugin.ts

```diff
@@ -129,14 +127,13 @@ interface ITimelineRecord {
 export interface IPrintTimelineParameters {
   terminal: ITerminal;
   result: IExecutionResult;
-  cobuildConfiguration: CobuildConfiguration | undefined;
 }
 
 /**
  * Print a more detailed timeline and analysis of CPU usage for the build.
  * @internal
  */
-export function _printTimeline({ terminal, result, cobuildConfiguration }: IPrintTimelineParameters): void {
+export async function _printTimeline({ terminal, result }: IPrintTimelineParameters): Promise<void> {
```

Comment:
Why async?

Reply:
Updated in [71336b2](https://github.com/microsoft/rushstack/pull/4680/commits/71336b2c606944bfda34f211a7af1b95ad67075b). This was a holdover from testing i was doing with getting the state file in this method instead of from operation execution record.